docs/datamodel/links.rst

```diff
@@ -24,3 +24,16 @@ The number of targets as specified by keywords :ref:`required
 and :ref:`multi <ref_eql_ddl_links_syntax>`.  It is also possible to
 restrict how many source objects can link to the same target via
 :eql:constraint:`exclusive` constraint.
+
+Links also have a policy of handling link target *deletion*. There are
+4 possible *actions* that can be taken when this happens:
+
+- ``RESTRICT`` - any attempt to delete the target object immediately
+  raises an exception
```

Comment:
Can you add ';' and finally a '.' in lists like this one?

Reply:
OK